src/utils/neynar/user/neynarUserFunctions.ts

```diff
@@ -0,0 +1,76 @@
+import { FetchError } from '../exceptions/FetchError';
+
+export const NEYNAR_DEFAULT_API_KEY = 'NEYNAR_API_DOCS';
+export interface NeynarUserModel {
+  fid: number;
+  custody_address: string;
+  username: string;
+  display_name: string;
+  pfp_url: string;
+  profile: {
+    bio: {
+      text: string;
+    };
+  };
+  follower_count: number;
+  verifications: string[];
+}
+export interface NeynarBulkUserLookupModel {
+  users: NeynarUserModel[];
+}
+
+export async function neynarBulkUserLookup(
```

Comment:
nit. for follow up PR, let's have more docs in here to explain what's going on.

Also let's try to keep the internal architecture as flat as possible, so that instead of having `src/utils/neynar/user/neynarUserFunctions.ts` we just have `src/utils/neynar/neynarUserFunctions.ts` as it will make it easier to understand for folks reading and contributing back.